container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -290,18 +290,32 @@ configuration:
       type: CACertificate
       value_type: private_key
     description: PEM-encoded JWT signing key.
-    required: true
   - name: KUBERNETES_CLUSTER_DOMAIN
+    description: >
+      The cluster domain used by Kubernetes.
+      If left empty, each container will try to determine the correct value based on /etc/resolv.conf
+      You can read more about it in the Kubernetes Documentation https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/
     type: environment
   - name: KUBERNETES_NAMESPACE
+    description: >
+      The name of the kubernetes namespace all components will run in.
+      This parameter cannot be set by the user.
+      Its value is supplied by the kubernetes runtime.
     type: environment
   - name: KUBE_COMPONENT_INDEX
-    type: environment
-  - name: KUBE_COMPONENT_NAME
+    description: >
+      This is an environment variable built-in by fissile.
+      It's set to a numeric index for roles with multiple replicas.
     type: environment
   - name: KUBE_SECRETS_GENERATION_COUNTER
+    description: >
+      This is an environment variable built-in by fissile.
+      It's automatically set to the kube.secrets_generation_counter Helm value, which controls secret rotation.
     type: environment
   - name: KUBE_SECRETS_GENERATION_NAME
+    description: >
+      This is an environment variable built-in by fissile.
+      Its default value is 'secret-1' and cannot be set by the user.
```

Comment:
This is not correct. The (dummy) `secrets-1` name is only used for kube-config files. In the helm chart the default value is:

```
const generatedSecretsName = "secrets-{{ .Chart.Version }}-{{ .Values.kube.secrets_generation_counter }}"
```